zokrates_book/src/toolbox/zokrates_js.md

```diff
@@ -123,12 +128,13 @@ const fileSystemResolver = (from, to) => {
 };
 ```
 
-##### computeWitness(artifacts, args)
+##### computeWitness(artifacts, args[, options])
 Computes a valid assignment of the variables, which include the results of the computation.
 
 Parameters:
 * `artifacts` - Compilation artifacts
 * `args` - Array of arguments (eg. `["1", "2", true]`)
+* `options` - Computation options
```

Comment:
The log stream is not mentioned here, should it be?

Reply:
I'm not sure about this one, I think most people won't touch this part, I added this just to have logs in the playground. It is possible to check the options if typescript is used though. The same lack of documentation is present for the compilation options too. I think the whole document should have a better structure, which can be improved in a separate PR.